loguru/_file_sink.py

```diff
@@ -17,12 +17,12 @@ def generate_rename_path(root, ext, creation_time):
     creation_datetime = datetime.datetime.fromtimestamp(creation_time)
     date = FileDateFormatter(creation_datetime)
 
-    renamed_path = "{}.{}{}".format(root, date, ext)
+    renamed_path = f"{root}.{date}{ext}"
```

Comment:
This is still valid, even in Python 3.13

May I ask what the scope of your PR is?
Are you simply trying to drop support for Python 3.5 and 3.6, or are you also resolving deprecated code and taking advantage of new syntax?

Reply:
This diff is generated by pyupgrade

I'm tring to resolving deprecated code and taking advantage of new syntax. and try add mypyc and cython on it. but py35 is just too old to do that.